package.json

```diff
@@ -49,6 +49,7 @@
     "react-copy-to-clipboard": "5.0.1",
     "react-dom": "16.8.6",
     "react-hot-loader": "4.12.15",
+    "react-is": "16.12.0",
```

Comment:
Is this intentional. The installation guide of `styled-components` doesn't mention `react-is`

Reply:
It's a peer dependency of styled-components and is mentioned here: https://github.com/styled-components/styled-components#upgrading-from-v4